ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-telemetry-postcard.j2

```diff
@@ -0,0 +1,72 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - monitor telemetry postcard #}
+{% if monitor_telemetry_postcard is arista.avd.defined %}
+!
+monitor telemetry postcard policy
+{%     if monitor_telemetry_postcard.disabled is arista.avd.defined(false) %}
+   no disabled
+{%     endif %}
+{%     if monitor_telemetry_postcard.ingress is arista.avd.defined %}
+{%         if monitor_telemetry_postcard.ingress.collection.source is arista.avd.defined and monitor_telemetry_postcard.ingress.collection.destination is arista.avd.defined and monitor_telemetry_postcard.ingress.collection.version is arista.avd.defined(2) %}
+   ingress collection gre source {{ monitor_telemetry_postcard.ingress.collection.source }} destination {{ monitor_telemetry_postcard.ingress.collection.destination }} version 2
+{%         elif monitor_telemetry_postcard.ingress.collection.source is arista.avd.defined and monitor_telemetry_postcard.ingress.collection.destination is arista.avd.defined %}
+{%             if monitor_telemetry_postcard.ingress.collection.version is arista.avd.defined(1) %}
+   ingress collection gre source {{ monitor_telemetry_postcard.ingress.collection.source }} destination {{ monitor_telemetry_postcard.ingress.collection.destination }}
+{%             else %}
+   ingress collection gre source {{ monitor_telemetry_postcard.ingress.collection.source }} destination {{ monitor_telemetry_postcard.ingress.collection.destination }}
+{%             endif %}
+{%         endif %}
+{%     endif %}
+{%     if monitor_telemetry_postcard.ingress.sample.rate is arista.avd.defined %}
+   ingress sample rate {{ monitor_telemetry_postcard.ingress.sample.rate }}
+{%     endif %}
+{%     if monitor_telemetry_postcard.ingress.sample.tcp_udp_checksum is arista.avd.defined %}
+   ingress sample tcp-udp-checksum {{ monitor_telemetry_postcard.ingress.sample.tcp_udp_checksum }}
+{%     endif %}
+{%     if monitor_telemetry_postcard.profiles is arista.avd.defined %}
+{%         for profile in monitor_telemetry_postcard.profiles %}
+   profile {{ profile.name }}
```

Comment:
need a check for the `name` property like you have in line 40

Reply:
added check for the key.